src/services/Models.service.tsx

```diff
@@ -295,3 +295,91 @@ export const triggerCallback = (callback?: () => void) => {
     callback();
   }
 };
+
+export const expressions = {
+  blink: [
+    {
+      morphTarget: 'eyesClosed',
+      morphTargetIndex: -1,
+      offset: 0,
+      duration: 0.2
+    },
+    {
+      morphTarget: 'eyeSquintLeft',
+      morphTargetIndex: -1,
+      offset: 0,
+      duration: 0.2
+    },
+    {
+      morphTarget: 'eyeSquintRight',
+      morphTargetIndex: -1,
+      offset: 0,
+      duration: 0.2
+    }
+  ]
+};
+
+/**
+ * Animates avatars facial expressions when morphTargets=ARKit,Eyes Extra is provided with the avatar.
+ */
+export const useIdleExpression = (expression: keyof typeof expressions, nodes: Nodes) => {
```

Comment:
do we want to add some warning if morphTarget is not present?

Reply:
Actually let-it-be for now